source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/RootViewController.swift

```diff
@@ -82,6 +91,11 @@ class RootViewController: NSViewController, NSTableViewDelegate, NSTableViewData
         guard selectedItem >= 0 && selectedItem < configs.count else { return config }
         let path = "/hostconfigs/\(configs[selectedItem])"
         let parts = path.components(separatedBy: ".")
+        if path.contains("dark") {
+            darkTheme = true
+        } else {
+            darkTheme = false
+        }
```

Comment:
```suggestion
        darkTheme = path.contains("dark")
```
Also, add a comment to explain the reason behind this.